src/dotnet/APIView/APIViewWeb/Repositories/CopilotCommentsRepository.cs

```diff
@@ -0,0 +1,50 @@
+using System.Threading.Tasks;
+using APIViewWeb.Models;
+using Microsoft.Azure.Cosmos.Serialization.HybridRow;
+using Microsoft.Extensions.Configuration;
+using MongoDB.Driver;
+using MongoDB.Driver.Linq;
+
+namespace APIViewWeb.Repositories
+{
+    public class CopilotCommentsRepository : ICopilotCommentsRepository
+    {
+        private readonly MongoClient _mongoClient;
+        private readonly string databaseName = "testDatabase";
```

Comment:
Shouldn't these be `const` instead of `readonly`?

Reply:
Just got rid of these fields since we can use user secrets instead.